client/state/selectors/get-site-computed-attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Would we ever really have a need for this selector in isolation from `getSite`, or should the implementation be baked into that selector instead? We only had need to extract logic for computing attributes to an independent function previously because both `lib/sites-list` and the Redux `getSite` selector were using it. Moving to the later exclusively seems a good reason to consolidate.